<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>bookin</class>
 <widget class="QWidget" name="bookin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>286</width>
    <height>281</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>bookin</string>
  </property>
  <widget class="QPushButton" name="But_BookIn">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>240</y>
     <width>80</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>入库</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>19</y>
     <width>251</width>
     <height>213</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label_ID">
      <property name="text">
       <string>ID:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_ISBN">
      <property name="text">
       <string>ISBN:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="lineEdit_ID"/>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="lineEdit_ISBN"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_name">
      <property name="text">
       <string>书名：</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="lineEdit_name"/>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_author">
      <property name="text">
       <string>作者：</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="lineEdit_author"/>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_publisher">
      <property name="text">
       <string>出版社：</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLineEdit" name="lineEdit_publisher"/>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="label_pubdate">
      <property name="text">
       <string>出版日期：</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QLineEdit" name="lineEdit_pubdate"/>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="label_score">
      <property name="text">
       <string>评分：</string>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QLineEdit" name="lineEdit_score"/>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>But_BookIn</sender>
   <signal>clicked()</signal>
   <receiver>bookin</receiver>
   <slot>Bookin()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>174</x>
     <y>252</y>
    </hint>
    <hint type="destinationlabel">
     <x>252</x>
     <y>259</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>Bookin()</slot>
 </slots>
</ui>
